dm/master/workerrpc/rawgrpc.go

```diff
@@ -124,6 +124,8 @@ func callRPC(ctx context.Context, client pb.WorkerClient, req *Request) (*Respon
 		resp.GetValidationError, err = client.GetValidatorError(ctx, req.GetValidationError)
 	case CmdOperateValidationError:
 		resp.OperateValidationError, err = client.OperateValidatorError(ctx, req.OperateValidationError)
+	case CmdUpdateValidation:
+		resp.UpdateValidation, err = client.UpdateValidator(ctx, req.UpdateValidation)
```

Comment:
the feature is related to cutover, but method named as `UpdateValidator`, sounds like update validator config online

Reply:
It's copied from spec. Any suggestion?